deploy/csv/redhat/bundle-1.8.0.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=percona-xtradb-cluster-operator-certified
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.2.0
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
+LABEL operators.operatorframework.io.metrics.project_layout=helm.sdk.operatorframework.io/v1
+LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
+LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
+
+#Add these labels
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```

Comment:
add `v4.7`?

Reply:
no mention here https://redhat-connect.gitbook.io/certified-operator-guide/appendix/bundle-maintenance-after-migration

Reply:
what does that mean? from what I see the tutorial is old and was created before 4.7 was released, what is the purpose of this LABEL field?